docs/topics/wildfly-swarm/docs/reference/fractions/hystrix.adoc

```diff
@@ -0,0 +1,120 @@
+= Hystrix
+
+
+.Maven Coordinates
+[source,xml]
+----
+<dependency>
+  <groupId>org.wildfly.swarm</groupId>
+  <artifactId>hystrix</artifactId>
+</dependency>
+----
+
+.Configuration
+
+swarm.hystrix.collapser.default.maxRequestsInBatch:: 
+The maximum number of requests allowed in a batch before this triggers a batch execution
+
+swarm.hystrix.collapser.default.requestCache.enabled:: 
+Indicates whether request caching is enabled for HystrixCollapser.execute() and HystrixCollapser.queue() invocations
+
+swarm.hystrix.collapser.default.timerDelayInMilliseconds:: 
+The number of milliseconds after the creation of the batch that its execution is triggered
+
+swarm.hystrix.command.default.circuitBreaker.enabled:: 
+Determines whether a circuit breaker will be used to track health and to short-circuit requests if it trips
+
+swarm.hystrix.command.default.circuitBreaker.errorThresholdPercentage:: 
+The error percentage at or above which the circuit should trip open and start short-circuiting requests to fallback logic
+
+swarm.hystrix.command.default.circuitBreaker.forceClosed:: 
+If true, forces the circuit breaker into a closed state in which it will allow requests regardless of the error percentage
+
+swarm.hystrix.command.default.circuitBreaker.forceOpen:: 
+If true, forces the circuit breaker into an open (tripped) state in which it will reject all requests
+
+swarm.hystrix.command.default.circuitBreaker.requestVolumeThreshold:: 
+The minimum number of requests in a rolling window that will trip the circuit
+
+swarm.hystrix.command.default.circuitBreaker.sleepWindowInMilliseconds:: 
+The amount of time, after tripping the circuit, to reject requests before allowing attempts again to determine if the circuit should again be closed
+
+swarm.hystrix.command.default.execution.isolation.semaphore.maxConcurrentRequests:: 
+The maximum number of requests allowed to a HystrixCommand.run() method when you are using ExecutionIsolationStrategy.SEMAPHORE
```

Comment:
same as above

Reply:
Note to self: not easily fixable in our sources.